sdk/monitor/azure-monitor-opentelemetry-autoconfigure/pom.xml

```diff
@@ -96,6 +96,11 @@
       <artifactId>opentelemetry-semconv-incubating</artifactId>
       <version>1.26.0-alpha</version> <!-- {x-version-update;io.opentelemetry.semconv:opentelemetry-semconv-incubating;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk-extension-incubator</artifactId>
+      <version>1.48.0-alpha</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-sdk-extension-incubator;external_dependency} -->
```

Comment:
Why are we taking a dependency on alpha version?

Reply:
It's for these OpenTelemetry semantic conventions:
* https://github.com/Azure/azure-sdk-for-java/blob/ff45b9d9a14e0e36ba1fb1333ebbfb405f015687/sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/AzureMonitorExporterBuilder.java#L105
* https://github.com/Azure/azure-sdk-for-java/blob/ff45b9d9a14e0e36ba1fb1333ebbfb405f015687/sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/AzureMonitorExporterBuilder.java#L106

Reply:
With an alpha dependency, this library should not release a GA version. We can use this dependency for preview releases.

Reply:
@jeanbisutti stable semconv we can use the stable semconv artifact: https://github.com/open-telemetry/semantic-conventions-java#published-releases

and for unstable semconv let's inline the constants